packages/dd-trace/src/config.js

```diff
@@ -1330,9 +1476,33 @@ class Config {
   // eslint-disable-next-line @stylistic/js/max-len
   // https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json
   _merge () {
-    const containers = [this._remote, this._options, this._env, this._calculated, this._defaults]
-    const origins = ['remote_config', 'code', 'env_var', 'calculated', 'default']
-    const unprocessedValues = [this._remoteUnprocessed, this._optsUnprocessed, this._envUnprocessed, {}, {}]
+    const containers = [
+      this._remote,
+      this._options,
+      this._fleetStableConfig,
+      this._env,
+      this._localStableConfig,
+      this._calculated,
+      this._defaults
+    ]
+    const origins = [
+      'remote_config',
+      'code',
+      'fleet_stable_config',
```

Comment:
Are these real values? Just want to make sure as I believe there is a schema for the origin.

Reply:
These values have been defined in the technical implementation RFC and will be the same across libraries. Is the schema in the backend or in this repository?

Reply:
In the backend, but if the values are defined by the RFC I would expect the schema to include them 👍

Reply:
It doesn't [seem to include them yet](https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v2/SchemaDocumentation/Schemas/conf_key_value.md). Since @paullegranddc is attached to the RFC I'd expect this'll happen sooner rather than later.